dashboard/app/models/applab.rb

```diff
@@ -205,7 +210,7 @@ def self.palette
         "functionParams_n": null,
         "callMyFunction": null,
         "callMyFunction_n": null,
-        "return": null
+        "return": null,
```

Comment:
Yeah, that sounds right. I've never really understood why JSON cares about them.